pkgs/development/libraries/science/chemistry/harminv/default.nix

```diff
@@ -30,7 +30,10 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ blas lapack ];
 
-  configureFlags = [ "--enable-shared" ];
+  configureFlags = [
+    "--enable-shared"
+    "--enable-maintainer-mode"
```

Comment:
What is the motivation for turning on maintainer-mode (Should be mentioned in the commit of PR message).

Reply:
Yeah, it's not necessary. For a dist phase it puts some copyright files in the tarball. I can disable it again.